arbitrator/prover/src/wavm.rs

```diff
@@ -267,6 +269,7 @@ impl Opcode {
             Opcode::ReadPreImage => 0x8020,
             Opcode::ReadInboxMessage => 0x8021,
             Opcode::HaltAndSetFinished => 0x8022,
+            Opcode::ReadHotShotCommitment => 0x9001,
```

Comment:
Is this just some arbitrary value?

Reply:
Yes, but should keep align with the solidity side